src/jestStandardReporter.js

```diff
@@ -75,7 +75,7 @@ class StandardReporter {
         testResult.snapshot,
         didUpdate
       );
-      snapshotStatuses.forEach(this.stdlib.log);
+      snapshotStatuses.forEach(this.stdlib.log.bind(this.stdlib);
```

Comment:
Missing a closing bracket?

Reply:
I'll take care of this.

Reply:
Thank you! sorry I couldn't do it in time.